storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3162,6 +3162,11 @@ static int rocksdb_compact_column_family(
   return HA_EXIT_SUCCESS;
 }
 
+static bool rocksdb_is_supported_system_table(const char *, const char *,
+                                              bool is_sql_layer_system_table) {
+  return is_sql_layer_system_table;
```

Comment:
I don't think this is safe: this allows `ALTER TABLE` for completely unsupported tables, and `ALTER TABLE` back to InnoDB would not be guaranteed. If we want to have the handlerton method anyway this early, then let's start with `return false` and enable the tables individually or in groups as the relevant work is done.